osbs/conf.py

```diff
@@ -43,14 +49,15 @@ def __init__(self, conf_file=DEFAULT_CONFIGURATION_FILE, conf_section=DEFAULT_CO
         :param kwargs: keyword arguments, which have highest priority: key is cli argument name
         """
         self.scp = configparser.SafeConfigParser()
-        try:
-            self.scp.read(conf_file)
-        except (IOError, TypeError):
-            pass
-        else:
-            if not self.scp.has_section(conf_section):
-                raise OsbsException("Specified section '%s' not found in '%s'" %
-                                    (conf_section, conf_file))
+        if conf_file and os.path.isfile(conf_file) and os.access(conf_file, os.R_OK):
+            try:
```

Comment:
I'd suggest removing the try-catch block as we're not expecting `self.scp.read()` to fail now, or do we?